cmd/azure/azure.go

```diff
@@ -17,7 +19,7 @@ var gitCommit = "unknown"
 
 func main() {
 	if err := run(); err != nil {
-		panic(err)
+		os.Exit(1)
```

Comment:
```log.Error(err)
os.Exit(1)``` Or is it intentional to drop error message?

Reply:
Yes - cobra already prints it out, we were duplicating it.